buf.gen.pyi.yaml

```diff
@@ -0,0 +1,4 @@
+version: v1
+plugins:
+  - remote: buf.build/jeffsawatzky/plugins/mypy-protobuf
```

Comment:
Hmm, when you check the [website for the plugin](https://buf.build/jeffsawatzky/plugins/mypy-protobuf), it seems that the custom remote plugins will be discontinued at some point which is a shame since this is a nice thing to have... However, a quick glance at the [plugins page](https://buf.build/plugins) reveals that there seems to be an official plugin for `.pyi` files - https://buf.build/protocolbuffers/pyi. Try to check this out, I'd rather use this official once since the other one seems like it will be discontinued.

Reply:
used the official plugin